$OpenBSD: patch-configure,v 1.9 2013/10/15 02:18:17 jeremy Exp $

Remove configure check, since the pl/python does work if you
LD_PRELOAD libpthread when starting PostgreSQL.

--- configure.orig	Thu May 15 21:53:18 2014
+++ configure	Thu May 15 21:53:51 2014
@@ -7356,11 +7356,6 @@ pythreads=`${PYTHON} -c "import sys; print(int('thread
 if test "$pythreads" = "1"; then
   { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
 $as_echo "yes" >&6; }
-  case $host_os in
-  openbsd*)
-    as_fn_error $? "threaded Python not supported on this platform" "$LINENO" 5
-    ;;
-  esac
 else
   { $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
 $as_echo "no" >&6; }
